types/markdown-to-jsx/index.d.ts

```diff
@@ -15,10 +16,14 @@ export interface MarkdownProps extends React.HTMLAttributes<HTMLElement> {
     children?: React.ReactNode;
 }
 
-export type ComponentOverride = string | React.ComponentClass | React.SFC | {
-    component: string | React.ComponentClass | React.SFC;
-    props?: any;
-};
+export type ComponentOverride<P = any, S = any> =
+    | string
+    | React.ComponentClass<P, S>
+    | React.FunctionComponent<P>
```

Comment:
Might be best to stick with SFC instead of FunctionComponent for now to avoid breaking compatibility with earlier React versions.

Reply:
`SFC` is deprecated but I agree this might be a good idea for people using old versions of React... that being said, apparently it’s good practice to switch to `React.FunctionComponent`. Both options work for me.

See https://stackoverflow.com/questions/53885993/react-16-7-react-sfc-is-now-deprecated

Reply:
It’s true that removing it would be good practice in private code, or code where you know all the consumers are on a certain minimum React version. But for “library” type code like this, it’s better to preserve backwards compatibility for awhile.

Reply:
Morning @ecraig12345. Just replaced `FunctionComponent` by `SFC`. All good to merge?